python/py-tensorflow/Portfile

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-tensorflow
+version             1.4.1
+platforms           darwin
+license             Apache-2
+maintainers         @emcrisostomo openmaintainer
+
+description         TensorFlow helps the tensors flow
+long_description    ${description}
+
+homepage            https://www.tensorflow.org/
+
+if {${python.version} eq 27} {
+    master_sites        https://pypi.python.org/packages/eb/40/0c425a97870f0e95a9d79e4fd5c2f609ee89a7e69b4644579d3157ed5ea0
```

Comment:
Same question as for py-tensorflow-tensorboard, does using `pypi:t/tensorflow` not work? That would also have the advantage of not having to be set differently for each subport.

Reply:
See above: I tried but no luck with the `whl` file